src/briefcase/platforms/iOS/xcode.py

```diff
@@ -421,19 +440,9 @@ def run_app(self, app: BaseConfig, udid=None, **kwargs):
                 "[{app.app_name}] Following simulator log output (type CTRL-C to stop log)...".format(app=app)
             )
             self.logger.info("=" * 75)
-            self.subprocess.run(
-                [
-                    "xcrun", "simctl", "spawn", udid,
-                    "log", "stream",
-                    "--style", "compact",
-                    "--predicate", 'senderImagePath ENDSWITH "/{app.formal_name}"'.format(app=app)
-                ],
-                check=True,
-            )
-        except subprocess.CalledProcessError:
-            raise BriefcaseCommandError(
-                "Unable to start log stream for app {app.app_name}.".format(app=app)
-            )
+            stream_popen_output(simulator_log_popen, self.logger.info)
+        except OutputStreamingError as e:
+            raise BriefcaseCommandError(f"Unable to start log stream for app {app.app_name}: {e}")
```

Comment:
To my reading, there's a couple of conditions other than "unable to start" that could trigger this case; this means we either need a more errors, or a more generic error.

Reply:
That's true....although, I changed this up and expounded in the next post.